src/components/sideBar/sideBar.jsx

```diff
@@ -0,0 +1,41 @@
+import { Link } from 'react-router-dom'
+
+import "./sideBar.css";
+import {
+    history,
+    home,
+    like,
+    playlist,
+    watchLater,
+} from "../../assets/icons";
+
+export function SideNavBar() {
+  const navLinks = [
+    {name: "Home", icon: home, link:"/"},
+    { name: "Playlist", icon: playlist, link:"/playlist" },
+    { name: "Watch Later", icon: watchLater, link:"/watchlater" },
+    { name: "Liked", icon: like, link:"/liked" },
+    { name: "History", icon: history, link:"/history" }
+    ];
+
+  return (
+    <aside className="side-navigation">
+      <ul>
+        {navLinks.map((navLink) => {
+          return (
+            <Link to={navLink.link}

```

Comment:
please provide key whenever you are using map function. If you dont have unique id property you can use. index as second parameter of map function as key

Reply:
Have made use of index parameter as suggested.